tests/storage/datavolume.go

```diff
@@ -955,6 +955,7 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 				}
 				err = virtClient.CoreV1().PersistentVolumeClaims(testsuite.NamespaceTestAlternative).Delete(context.Background(), dataVolume.Name, metav1.DeleteOptions{})
```

Comment:
how come we need to delete the pvc that is created by the dv? isnt it enough to wait for the pvc to be deleted?

Reply:
yep, good point, added a change